VideoLocker/res/values/dimens.xml

```diff
@@ -151,7 +151,9 @@
 
     <dimen name="user_profile_image_diameter">166dp</dimen>
     <dimen name="user_profile_image_border_width">3dp</dimen>
-    <dimen name="crop_image_guideline_width">3dp</dimen>
+
+    <dimen name="discussion_image_diameter">40dp</dimen>
+    <dimen name="discussion_image_border_width">1dp</dimen>
```

Comment:
This [doesn't seem][diff] to be used anymore.

[diff]: https://github.com/edx/edx-app-android/pull/752/commits/b8715ba15340e173662fac2dd8f5238d0b72e128#diff-d0a326a6c05d63875048ea9563fded72L297